src/main/java/reactor/netty/channel/BootstrapHandlers.java

```diff
@@ -489,6 +489,10 @@ public static Bootstrap removeMetricsSupport(Bootstrap b) {
 		return b;
 	}
 
+	/**
+	 * Changes the resolver to {@link NoopAddressResolverGroup} if the current one is
+	 * {@link DefaultAddressResolverGroup}.
+	 */
 	public static Bootstrap updateResolverForProxySupport(Bootstrap b) {
```

Comment:
I wonder if the name of the method shouldn't be changed to something more meaningful like `forceNoopResolver`, or something like that. Here the name makes me think it checks `Bootstrap` for proxy configuration. And also makes me think it sets up a meaningful `resolver`, not merely a `noop` one.

Reply:
It forces to `Noop...` only if there is no custom resolver. Also it takes care to update the resolver used when metrics are enabled. So I think `updateResolverForProxySupport` describes exactly what happens.

Reply:
well, if resolver is already noop, then the end result is the same ;) but fair enough.